core-bundle/src/Resources/contao/languages/en/modules.xlf

```diff
@@ -305,6 +305,12 @@
       <trans-unit id="FMD.rssReader.1">
         <source>Adds an RSS feed to the page</source>
       </trans-unit>
+      <trans-unit id="FMD.feed_reader.0">
```

Comment:
should remove `rssReader` a few lines above?

Reply:
Fixed in 243491c5f